crypto/asn1/x_long.c

```diff
@@ -56,11 +56,30 @@ static void long_free(ASN1_VALUE **pval, const ASN1_ITEM *it)
     *(long *)pval = it->size;
 }
 
+static int num_bits_ulong(unsigned long value)
```

Comment:
Is there a good reason to have two functions (this and BN_num_bits_word) that essentially do the exact same thing?